go/interpreter/lfvm/instructions.go

```diff
@@ -22,16 +22,23 @@ func opStop() status {
 	return statusStopped
 }
 
-func opRevert(c *context) status {
-	c.resultOffset = *c.stack.pop()
-	c.resultSize = *c.stack.pop()
-	return statusReverted
+func opRevert(c *context) (status, error) {
```

Comment:
I have seen this pattern before with call and create, it produces symbols and names that we may not need. 

Better rename  `genericReturnRevert` into `opReturningOp`  and call it in the switch:
```
      case RETURN:
		status, err = opReturningOp(c, statusReturned)
      case REVERT:
		status, err = opReturningOp(c, statusReverted)			
```
This change saves us from one line functions that do not contribute